src/util/expr.h

```diff
@@ -361,7 +361,7 @@ class exprt:public irept
 };
 
 /// Base class for all expressions. This protects low-level methods in
-/// exprt that are not type safe. Depcrecated constructors are removed.
+/// exprt that are not type safe. Deprecated constructors are removed.
```

Comment:
Although this can be understood as overhead: please put changes like these in a separate commit and PR. Great to see those fixes, but it's an entirely unrelated change.

Reply:
I'll add this change on a separate PR.